lib/Driver/ToolChains.cpp

```diff
@@ -1109,6 +1109,40 @@ ToolChain::constructInvocation(const StaticLinkJobAction &job,
    llvm_unreachable("archiving not implemented for this toolchain");
 }
 
+ToolChain::InvocationInfo
+ToolChain::constructInvocation(const LoadModuleJobAction &job,
+                               const JobContext &context) const {
+
+  // Write a temporary file that imports the module we're trying to pre-load.
+  auto tempFile = context.getTemporaryFilePath("tmp.swift");
+
+  std::error_code err;
+  llvm::raw_fd_ostream os(tempFile, err);
+  llvm::SmallString<32> importDecl("import ");
+  importDecl.append(job.getModuleName());
+  os << importDecl.str() << "\n";
```

Comment:
It's not really okay for the toolchain to write files during invocation construction. But we do have `-import-module Foo` as a hidden frontend option anyway (because Playgrounds uses it), so maybe we can just pass an empty file instead.